bugbear.py

```diff
@@ -243,7 +243,9 @@ def _flatten_excepthandler(node: ast.expr | None) -> Iterator[ast.expr | None]:
         yield expr
 
 
-def _check_redundant_excepthandlers(names: Sequence[str], node, in_trystar):
+def _check_redundant_excepthandlers(
+    names: Sequence[str], node: ast.ExceptHandler, in_trystar: str
```

Comment:
surely this?
```suggestion
    names: Sequence[str], node: ast.ExceptHandler, in_trystar: bool
```

Reply:
nope, I thought so too at first but it's from
https://github.com/PyCQA/flake8-bugbear/blob/826c071ce81b70562daa2cc7ba37cef1d5d74a05/bugbear.py#L391-L392